ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py

```diff
@@ -1403,7 +1403,15 @@ def test_torch_mul(
     fn_tree="torch.div",
     aliases=["torch.divide"],
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("numeric"),
```

Comment:
Could you change this to `available_dtypes =helpers.get_dtypes("valid")` and if there's a test failure due to dtype, add the unsupported dtype decorator appropriately?

Reply:
Thanks! I've gone the failing tests for test_pointwise_ops.py and test_tensor.py and added the unsupported dtypes.

Reply:
Okay great, could you please comment `add_frontend_checklist` , tick off the items there, and take a quick look to confirm that the failures have nothing to do with your code please. Will then review